### ISSUE TITLE
Deleting podspec line

### ISSUE THREAD

**Issue by NathanaelA** [ABTSoftware/PodSpecs3.x#1]
FYI: this line:
https://github.com/ABTSoftware/PodSpecs3.x/blob/master/SciChart/3.0.0.5098/SciChart.podspec#L15
Is causing podspec build failures when used with some other pod's.   The issue is that this seems to be setting the podfile to ios 8.   But if you have any other podspecs that requires 9; it will fail because the value is set to 8.      

Tried several work arounds based on StackOverflow posts, and couldn't find anything that worked.

Final Workaround:
I just locally cloned this spec file; deleted that line and pointed my podfile to my version of the spec, but I would much rather have it point to your spec repo so that it can stay updated....

**Comment by andyb1979** [comment]
Thanks Nathanael!

I’ll forward this on

**Comment by nrudnyk** [comment]
Hello Nathanael,

The line you mentioned sets deployment target, which is described [here](https://guides.cocoapods.org/syntax/podspec.html#deployment_target). It defines the minimum deployment targets of the supported platforms, in our case - iOS. 
So I'm not sure what's causing the issue on your side. That's something which might be related to [this](https://stackoverflow.com/questions/37160688/set-deployment-target-for-cocoapodss-pod). Hope that helps!

**Comment by NathanaelA** [comment]
I just figured it out as I was creating you a case to show you the issue...   :grinning:

Apparently when we add this:
```
source 'https://github.com/ABTSoftware/PodSpecs3.x.git'
pod 'SciChart'
```

ALL OTHER SOURCES are disabled...   Which then means the default Cocoapod source, no longer is active, meaning the next pod file fails...        So my "Workaround" just pointed the "pod" to my local podspec, which meant I didn't need a new `source` which is why it worked and gave me the impression the version number mattered.      Sorry about the false report...

---

So basically in your documentation where you mention putting in your source; you might want say something to the effect about this and change it to be:

```
source 'https://github.com/CocoaPods/Specs.git'
source 'https://github.com/ABTSoftware/PodSpecs3.x.git'
pod 'SciChart'
```

So that other pod files are NOT broken...
